examples/nextjs-presence/.env.production

```
@@ -1,6 +1,7 @@
 # Yorkie API Keys (Production)
 NEXT_PUBLIC_YORKIE_API_ADDR=https://api.yorkie.dev
 NEXT_PUBLIC_YORKIE_API_KEY=D6sd9E5ehtXBssmeRZ4QQP
+NEXT_PUBLIC_BASE_PATH=/yorkie-js-sdk/examples/nextjs-presence
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_BASE_PATH key should go before the NEXT_PUBLIC_YORKIE_API_ADDR key

(UnorderedKey)